### ISSUE TITLE
Panic: Index out of range 

### ISSUE THREAD

**Issue by c-iyer** [hideo55/go-loudstrie#1]
Hi, 
I am trying to use the trie package for a research project. I was able to build the trie from a text file containing US location strings. 
When I try to do an ExactMatchSearch() however it fails with 

```
panic: runtime error: index out of range

goroutine 16 [running]:
runtime.panic(0x7445a0, 0xae969c)
        /usr/lib/golang/src/pkg/runtime/panic.c:279 +0xf5
github.com/hideo55/go-sbvector.(*BitVectorData).Select1(0xc20802a0e0, 0x8143, 0x10200, 0x0, 0x0)
        /home/go/src/github.com/hideo55/go-sbvector/sbvector.go:447 +0x22a
github.com/hideo55/go-loudstrie.(*TrieData).getChild(0xc20803c000, 0x8161, 0xc2083ade30, 0xc2083ade28)
        /home/go/src/github.com/hideo55/go-loudstrie/loudstrie.go:269 +0xc4
github.com/hideo55/go-loudstrie.(*TrieData).Traverse(0xc20803c000, 0x8347f0, 0x16, 0x16, 0xc2083ade30, 0xc2083ade28, 0xc2083ade38, 0xffffffffffffffff, 0x1)
        /home/go/src/github.com/hideo55/go-loudstrie/loudstrie.go:235 +0x261
github.com/hideo55/go-loudstrie.(*TrieData).ExactMatchSearch(0xc20803c000, 0x8347f0, 0x16, 0x17, 0x0)
        /home/go/src/github.com/hideo55/go-loudstrie/loudstrie.go:128 +0x96
github.com/jackmobile/one/ml.(*Trie).Contains(0xc208442a40, 0x8347f0, 0x16, 0x6)
        /home/go/src/github.com/jackmobile/one/ml/trie.go:43 +0xf2
main.main()
        /home/go/src/github.com/jackmobile/one/tmp1.go:16 +0x108
```

The string I am trying to search for is "rancho santa margarita"

**Comment by hideo55** [comment]
This problem fixed in b0f1e747f74504f3eec50681602c262cc97dca5e.(Bug in loudstrie.Build() and sbvector.Select())
Please update `go-loudstrie` and `go-sbvector`.

```
go get -u github.com/hideo55/go-sbvector
go get -u github.com/hideo55/go-loudstrie
```

**Comment by c-iyer** [comment]
Thanks for the fix. It appears to be working now.
